### DISCUSSION TITLE
Cambio de nombre de algunas clases

### DISCUSSION THREAD

**Discussion by JavierMarrero** [JavierMarrero/jlibgraph#46] [General]
@SnowBlackQueen el nombre de la clase que contiene el algoritmo de Kosaraju debe ser un poco más descriptivo.
Puede ser `KosarajuAlgorithm` o algo similar. Lo mismo para el algoritmo de Dial, recomiendo `DialShortestPath`.

@cdrobaina01 `ExistPath` también debe ser más descriptiva.

**Comment by SnowBlackQueen** [comment]
vale vale. en el próximo pull request subo ese cambio 😅

**Reply by JavierMarrero** [reply]
Ya ya vi que si 😁

**Comment by JavierMarrero** [comment]
@cdrobaina01 ya @SnowBlackQueen cumplió, me faltas tu.
